lib/webauto/web4cucumber.rb

```diff
@@ -568,7 +568,7 @@ def handle_element(element_rule, **user_opts)
           res_join res, opres
         end
       end
-
+      take_screenshot
```

Comment:
This will take screenshot at every operation. This will be huge amount of screenshots.

Reply:
Yeah, I'm enabling it just temporarily for debugging, will remove it when ready to merge